Archipelago.MultiClient.Net/Packets/RoomUpdatePacket.cs

```diff
@@ -13,8 +13,5 @@ public class RoomUpdatePacket : RoomInfoPacket
 
         [JsonProperty("checked_locations")]
         public List<int> CheckedLocations { get; set; }
-
-        [JsonProperty("missing_locations")]
-        public List<int> MissingLocations { get; set; }
```

Comment:
Why remove this property? It is part of the RoomUpdatePacket as far as I can tell.

Reply:
No actually its not, 
https://github.com/ArchipelagoMW/Archipelago/blob/main/docs/network%20protocol.md#arguments-5
states "Should never be sent as an update, if needed is the inverse of checked_locations.", which basicly means you wont find it on an RoomUpdatedPacket ever

Wich is also confirmed here: https://discord.com/channels/731205301247803413/731214280439103580/926991639912788018

Reply:
This needs fixing in documentation to be more clear. Bleh. Thanks.

Reply:
Agreed it also confused me at first